functions.php

```diff
@@ -289,6 +289,34 @@ public function register_header_metabox() {
 				'preview_size' => 'large',
 			)
 		);
+
+		$p4_post = new_cmb2_box( array(
+			'id'           => $prefix . 'metabox_post',
+			'title'        => __( 'Post Articles Element Fields', 'planet4-master-theme' ),
+			'object_types' => array( 'post' ),
+		) );
+
+		$p4_post->add_field( array(
+			'name' => __( 'Articles Title', 'planet4-master-theme' ),
+			'desc' => __( 'Articles title comes here', 'planet4-master-theme' ),
+			'id'   => $prefix . 'articles_title',
+			'type' => 'text_medium',
+		) );
+
+		$p4_post->add_field(
+			array(
+				'name'            => __( 'Articles Count', 'planet4-master-theme' ),
+				'desc'            => __( 'Articles count comes here', 'planet4-master-theme' ),
```

Comment:
Perhaps more descriptive `desc` is needed for editors? I know I don't know GP data, but that desc doesn't really clarify anything.